clients/bigquery/converters/converters.go

```diff
@@ -25,6 +26,15 @@ func (s StringConverter) Convert(value any) (any, error) {
 		return castedValue.String(), nil
 	case bool, int64:
 		return fmt.Sprint(castedValue), nil
+	case time.Time:
```

Comment:
Want to add "time.Time" to the error message returned in the default case below?

Reply:
Let me modify the error message since it's getting long